spec/openapi.yaml

```diff
@@ -624,137 +611,188 @@ components:
           type: boolean
           example: true
 
-    FileInfo:
+    FileTreeObject:
       type: object
       properties:
-        type:
-          type: string
-          description: File could be Print file, firmware or other (e.g. configuration file)
-          enum: ["PRINT_FILE", "FIRMWARE", "FILE", "FOLDER"]
-        refs:
-          type: object
-          properties:
-            download:
-              type: string
-              example: "/api/files/local/examples/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode/raw"
-            icon:
-              type: string
-              example: "/api/thumbnails/local/examples/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode.small.png"
-            thumbnail:
-              type: string
-              example: "/api/thumbnails/local/examples/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode.orig.png"
-        size:
-          type: integer
-          example: 424242
-        display_path:
+        name:
           type: string
-          example: "/PrusaLink gcodes/examples"
+          example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
         display_name:
           type: string
           description: Long Filename
           example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
-        creation_timestamp:
+        path:
+          type: string
+          example: "/local/Dune/Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        display_path:
```

Comment:
We have to discuss that one more time in our internal Slack channel. It bad, if one path contains file_name and another not.

Reply:
I think it's okay to remove filename from the both paths, so we'll have `/local/Dune` and `/PrusaLink gcodes/Dune`